lib/utils/compose.coffee

```diff
@@ -230,7 +230,7 @@ exports.buildProject = (
 					containerQemuPath: "/tmp/#{qemu.QEMU_BIN_NAME}",
 					qemuFileMode: 0o555,
 				},
-				dockerfilePath
+				dockerfilePath or undefined,
```

Comment:
A recent PR (#1628) introducing the feature of specified/alternative Dockerfile names in the context of QEMU emulation resulted in the `dockerfilePath` value on this line being the empty string (instead of undefined) when a name is **not** specified on the command line (i.e., when the `--dockerfile` option is **not** used).
But the [transposeTarStream function](https://github.com/balena-io-modules/docker-qemu-transpose/blob/v1.0.3/src/index.ts#L213) expects the value to be `undefined` (rather than the empty string) in order to use the default `'Dockerfile'` name, hence adding `or undefined` here.

I have started writing test cases to cover the `build --emulation` functionality, but it's taking some time to get right and this fix is relatively urgent. So I'd like to get this PR through without the automated test cases, and I'll add the tests cases in a subsequent PR.